client/public/index.html

```diff
@@ -2,42 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Let's keep the favicon.  You would need to bring the file back in that you deleted.